contracts/Vault.vy

```diff
@@ -791,7 +791,7 @@ def withdraw(_shares: uint256 = MAX_UINT256, _recipient: address = msg.sender) -
     log Transfer(msg.sender, ZERO_ADDRESS, shares)
```

Comment:
Wouldn't you want to assert this also?

Reply:
logs appear in the transaction receipt. broadly, we should probably test for event logs occurring correctly, but haven't gotten to that yet.

Reply:
How about changing this to
```suggestion
    assert self.transfer(ZERO_ADDRESS, shares)
```
?

Reply:
And if I remember correctly there's another place where `log Transfer` is used and this could also be done

Reply:
Ah, I think I see the confusion. This function is taking place of a mint function, it doesn't actually transfer any shares but creates them. This is a weird aspect of using ERC20.

Reply:
Ah, right. That somewhat makes sense